e2e/ruler_test.go

```diff
@@ -116,7 +116,7 @@ func createRuleEvaluatorOperatorConfig(ctx context.Context, t *OperatorContext,
 				Alertmanagers: []monitoringv1.AlertmanagerEndpoints{
 					{
 						Name:       "test-am",
-						Namespace:  t.namespace,
+						Namespace:  t.userNamespace,
```

Comment:
hm, or namespace (depends if we test managed or unmanaged?)

Reply:
Good catch but that is not how managed alertmanager is configured! It looks like we automagically insert managed-alertmanager into the YAML configuration if we find the service (in `operator_config.go`; see the beginning of `makeAlertmanagerConfigs`.